getting-started/README.md

```diff
@@ -72,6 +72,10 @@ Compiling a native executable takes a bit longer, as GraalVM performs additional
 steps to remove unnecessary codepaths. Use the  `native` profile to compile a
 native executable:
 
+To run on a non-Linux system (GraalVM and Docker required):
+> ./mvnw clean install -Pnative -Dnative-image.docker-build=true
```

Comment:
It's not totally true.

```
./mvnw install -Dnative
```

works on all systems, but the resulting executable is specific to the host. So except on Linux, you cannot copy that file into a container. 

I would extend the documentation and add a container section.

Reply:
/me still thinking having these READMEs there is a bad idea and they should simply point to the guide where we would centralize everything.

Reply:
I agree! I can't remember why we added them at the beginning (maybe it was before we had any doc... good old times :-D )

Reply:
Thanks for answering @cescoffier and @gsmet! Apologies for the delay in my answer.  

I also appreciate the clarification regarding the following command:
`./mvnw install -Dnative`

Extending to the documentation to add a container section seems the best option. I can take a further look at the Quarks documentation.

Reply:
Regarding building with ./mvnw install -Dnative, it is not compiling (see my post https://quarkusio.zulipchat.com/#narrow/stream/187030-users/topic/Native.20build.20via.20Docker.20fails/near/219348056). I got Ubuntu 20.04 and jdk11. The same compilation problems occurs with only ./mvnw install -Dnative.

Reply:
Interesting, thanks for the comment @bartwinski! Should new Quarkus users need to go deep into those matters or should Quarkus have a command that works right away? Maybe adding more detailed documentation in the README file would be useful. 

Alternatively, adding a command that just works would make other developer's life easier.